modules/face/samples/facemark_demo_lbf.cpp

```diff
@@ -63,6 +66,14 @@ int main(int argc, char** argv)
     FacemarkLBF::Params params;
     params.model_filename = model_path;
     params.cascade_face = cascade_path;
+    /* Change given parameters according to need. (if not given by user, set by default - default arguments given below).
+    // FacemarkLBF supports 29 and 68 point landmarks currently.
```

Comment:
unfortunately, as it is now, the lbf model does not support any other than 68 points

please [see the answer to your question here](http://answers.opencv.org/question/192167/facemarklbf-training-fewer-landmark-points-than-68/)